light/proxy/routes.go

```diff
@@ -136,17 +136,17 @@ func makeCommitFunc(c *lrpc.Client) rpcCommitFunc {
 
 type rpcDataCommitmentFunc func(
 	ctx *rpctypes.Context,
-	beginBlock uint64,
-	endBlock uint64,
+	minHeight int64,
+	maxHeight int64,
 ) (*ctypes.ResultDataCommitment, error)
 
 func makeDataCommitmentFunc(c *lrpc.Client) rpcDataCommitmentFunc {
 	return func(
 		ctx *rpctypes.Context,
-		beginBlock uint64,
-		endBlock uint64,
+		minHeight int64,
```

Comment:
Thanks a lot for taking care of this :+1: 

I've started working on renaming `beginBlock` and `endBlock` to `firstBlock` and `lastBlock`. If we decide to go for `minHeight` and `maxHeight`, then I guess we should update this issue too https://github.com/celestiaorg/celestia-core/issues/928 and create new ones for the new PRs that will be merged.

Thus, I guess that if we leave this PR until we merge the inclusion proofs ones and rename everything at the same time, it will be easier. I am okey with either anyway. Whichever you prefer.

Reply:
Also, for the `uint64` -> `int64`, If we change that here, I guess to stay consistent, we will need to do it for the QGB module in celestia-app as well.

Reply:
I don't have strong preferences on the name. This PR uses `minHeight` because that was proposed in https://github.com/celestiaorg/celestia-core/pull/898#discussion_r1038012583

> Also, for the uint64 -> int64, If we change that here, I guess to stay consistent, we will need to do it for the QGB module in celestia-app as well.

That seems reasonable to me